.travis.yml

```diff
@@ -1,10 +1,14 @@
+---
+
 language: python
 python:
-  - "2.6"
   - "2.7"
-  - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.9-dev"
```

Comment:
Should perhaps drop this as `3.9-dev` will eventually go away after `3.9` is released.

Reply:
Dropped in 47fc333